lib/class-experimental-wp-widget-blocks-manager.php

```diff
@@ -245,6 +257,9 @@ public static function serialize_blocks( $blocks ) {
 	 * @return string String representing the block.
 	 */
 	public static function serialize_block( $block ) {
+		if( ! isset( $block[ 'blockName' ] ) ) {
```

Comment:
Spacing is not quite right here. 

```suggestion
		if ( ! isset( $block['blockName'] ) ) {
```